src/packer.json

```diff
@@ -30,13 +30,13 @@
       "skip_create_ami": "{{user `skip_create_ami`}}",
       "source_ami_filter": {
         "filters": {
-          "name": "debian-10-amd64-*",
+          "name": "debian-11-amd64-*",
           "root-device-type": "ebs",
           "virtualization-type": "hvm"
         },
         "most_recent": true,
         "owners": [
-          "136693071363"
+          "679593333241"
```

Comment:
Why are we changing the AMI owner here?  According to [this](https://wiki.debian.org/Cloud/AmazonEC2Image/Bullseye), the bullseye AMIs are available from `136693071363`.

Reply:
I saw those images.  I also checked in Amazon Marketplace and saw some ["Official Debian 11" AMIs](https://console.aws.amazon.com/marketplace/home?region=us-east-1#/subscriptions/a264997c-d509-4a51-8e85-c2644a3f8ba2) for which we could subscribe.  I subscribed, which gave us access to `ami-0a943bffbae98ca20` from owner `679593333241`.

I'll change it back.  I wasn't sure which AMIs were the right ones, but your link makes it quite clear.

Reply:
See commit f6edab0.

Reply:
I believe the account number `679593333241` is for AMIs offered by the AWS Marketplace. I believe they would be identical to those offered directly by the Debian account number, but they are considered a different owner as they are republished. Please see [this comment](https://debian-cloud.debian.narkive.com/kbtLpX5n/stretch-ec2-information-doesn-t-appear-to-be-correct#post2) as the source I used to verify that belief.

Reply:
Nice detective work Mr. Holmes!